include/samplefile.hpp

```diff
@@ -46,9 +46,11 @@ class SampleFile {
         }
         void writeToFile(char* line) {
             lock.lock();
-            rte_memcpy(_mmap + *_lastpos,
-                line,
-                MAX_BUFSIZE);
+            char* src = line;
+            char* dst = _mmap + *_lastpos;
+            for(int i = 0; i < MAX_BUFSIZE; i++) {
```

Comment:
Why is so much data being copied? I would have thought it would just be copying the length of the string.

Reply:
This is the parameterization used in your original code! There's garbage at the end of the signal files that I think is the result of this.

Reply:
Ha! Awesome. Might as well not perpetuate the bug I guess :).

Reply:
OK, will merge and fix later.